api/datastore/sql/migrations/1_add_route_created_at.go

```diff
@@ -0,0 +1,27 @@
+package migrations
+
+import (
+	"database/sql"
+	"github.com/pressly/goose"
+)
+
+// we skip the error here to make previous datastore tables work fine
```

Comment:
this comment line is in every one of these migration files, as well as the README here. a simple sed should get rid of them all.

Reply:
fixed